content/en/platform/corda/4.10/enterprise/operations/monitoring-logging/monitoring-logging-intro.md

```diff
@@ -47,7 +47,7 @@ To understand the information in the logs, which data is important, and what can
 
 ## Monitoring - the basics
 
-You can monitor the health and performance of your node [using tools such as Jolokia and Graphite](../../node/operating/monitoring-logging/overview.html#node-monitoring). Whenever using these tools, you should follow best-practice steps such as setting up TCP checks for the ports the node communicates on, database health checks etc.
+You can monitor the health and performance of your node [using tools such as Jolokia and graphite. Whenever using these tools, you should follow best-practice steps such as setting up TCP checks for the ports the node communicates on, database health checks etc.
```

Comment:
Think this should be upper case even if their logo is stylised as lower case?

Reply:
Hmm, not sure, if you scroll down the page, it's lower case in our docs so I wanted to stay consistent.